### DISCUSSION TITLE
Reposaur for GitLab

### DISCUSSION THREAD

**Discussion by abhi4u1947** [reposaur/reposaur#38] [Q&A] [answered]
@crqra Will, it be possible to get some support from GitLab Community to work on this and make it available for GitLab

**Comment by crqrdotcom** [comment, marked as answer]
Hey @abhi4u1947 👋🏽  - thank you for your interest in Reposaur and Gitlab integration. I've answered a similar question [here](https://github.com/reposaur/reposaur/discussions/36#discussioncomment-3688683). Let's continue the discussion there so it doesn't get fragmented.

Thank you!

**Reply by crqrdotcom** [reply]
Related issue https://github.com/reposaur/reposaur/issues/41
